sdk/storage/azure-storage-blobs/test/ut/CMakeLists.txt

```diff
@@ -32,6 +32,7 @@ add_executable (
     macro_guard.cpp
     page_blob_client_test.cpp
     page_blob_client_test.hpp
+    simplified_header.cpp
```

Comment:
Is there a reason to break from the normal pattern of ending test files with *_test.[hpp|cpp]?

Reply:
Oops, it's a mistake. I'll fix it